### ISSUE TITLE
Move authorities to utils class

### ISSUE THREAD

**Issue by cssru** [php-coder/mystamps#352]
Move hard coded authorities from CustomUserDetailsService class to utilities class.

**Comment by php-coder** [comment]
Why do we need this change? Why do you think that it's optimization? How it makes code faster?

**Comment by cssru** [comment]
We want to avoid magic on code, right?

**Comment by cssru** [comment]
May be, it is better to use Enum?

**Comment by php-coder** [comment]
> We want to avoid magic in code, right?

Yes, we're, but I don't see magic here. I did it intentionally: 1) I put the application specific information close to the place where we're using it 2) it belongs to the package with Spring Security related classes

Probably, we would extract this method to another class only when there will be another piece of code that's doing the same things (but I can't imagine such case).

In other words, my opinion that this code placed to the right place and don't need refactoring. This code doesn't look like utils, because it's very specific to the application, it's used only once in the application and most likely it won't be used somewhere else.

Are you agree? What's your opinion on that?

**Comment by php-coder** [comment]
> May be, it is better to use Enum?

What benefit we will have in this case? +1 more class and that's all? I will agree with you if you show me other places where this enum could be used (I know about Thymeleaf templates, but I don't know how to use this enums there).

**Comment by cssru** [comment]
Hm...and what about service package?

**Comment by php-coder** [comment]
>Hm...and what about service package?

I don't understand to what of my comment you posted your comment :) You mean that somewhere is service package we could use enum with authorities? Where? Annotation expect to have a string, and AFAIR it won't accept enum (you can try it yourself).

**Comment by php-coder** [comment]
I'm going to close this issue as `wontfix` unless you convince me do not doing it.
